src/silverlabnwb/nwb_file.py

```diff
@@ -488,8 +493,8 @@ def determine_trial_times(self):
             trial = 'trial_{:04d}'.format(i + 1)
             self.nwb_file.add_epoch(
                 name=trial,
-                start_time=start_time if i == 0 else np.nextafter(start_time, stop_time),
-                stop_time=np.nextafter(stop_time, stop_time * 2),
+                start_time=start_time if i == 0 else start_time + 1e-9,
+                stop_time=stop_time + 1e-9,
```

Comment:
I think `nextafter` should be fine here, as we just want to make sure we include the measurement at exactly `stop_time`. Perhaps it's better for consistency though!

Reply:
reverted for now, as we don't really have a reason to make the change?